test/functional/feature_block.py

```diff
@@ -598,77 +577,78 @@ def get_tests(self):
         self.tip = b46
         assert 46 not in self.blocks
         self.blocks[46] = b46
-        yield self.rejected(RejectResult(16, b'bad-blk-length'))
+        # s = ser_uint256(b46.hashMerkleRoot)
```

Comment:
In commit 06d512f11998d9b935edf8b032eb76e491cfb7cc:

Why are you adding this comment? The `ser_uint256` symbol is not imported and it will unsurprisingly print all zeros, since it was set to 0.

Reply:
removed